Makefile

```diff
@@ -25,7 +25,7 @@ SMF_MANIFESTS_IN = smf/manifests/registrar.xml.in
 # RELENG-341: no npm cache is making builds unreliable
 NPM_FLAGS :=
 
-NODE_PREBUILT_VERSION=v0.10.48
+NODE_PREBUILT_VERSION=v6.17.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Use sdcnode built for multiarch-15.4.1
```

Comment:
This is the part that'll change in theory once rabbitmq goes 64-bit?

Reply:
It's actually more down on line 32 where the base image will be changed to 19.4.0 (but that also necessitates changing the node version to 6.17.1).